app/models/course.rb

```diff
@@ -78,4 +78,16 @@ def open_for_applications?
   def has_vacancies?
     site_statuses.with_vacancies.any?
   end
+
+  def study_mode_description
+    study_mode.to_s.tr("_", " ")
+  end
+
+  def description
```

Comment:
It is worth breaking this down? It's really hard to read...

Reply:
I've taken a stab at making it more readable, have another look.